Packs/MISP/Integrations/MISPV3/MISPV3.py

```diff
@@ -302,6 +302,28 @@ def build_generic_object(template_name: str, args: list[dict]) -> GenericObjectG
     return misp_object
 
 
+def build_custom_object(template_name: str, args: list[dict]):
+    obj = PYMISP.object_templates()
+    for ob in obj:
```

Comment:
Let's try to think of a more meaningful name, maybe `for entry in obj`

Reply:
Thanks for the feedback. Adjusted to entry with the latest commit.